src/components/NavigationPanel/Metric/Metric.tsx

```diff
@@ -1,9 +1,14 @@
-"use client";
```

Comment:
Add this back. It is an important indicator to the Next.js bundler that this component can only be used as a client component.